README.md

```diff
@@ -38,6 +38,16 @@ docker run -it --rm --mount "type=bind,source=$(pwd),target=/code" \
     -p 4000:4000 spinnaker/spinnaker.github.io-test --incremental
 ```
 
+## Host the website on Amazon S3
```

Comment:
Why is this section necessary? Is there something you're trying to do that isn't covered by the local development steps?

cc @dorbin

Reply:
Local development necessarily does not gives way to review the changes. Let say, i made changes and i want it to be reviewed by someone (not only the content, but also navigational look and feel) , this way paves way for it.

Reply:
@anshrma 
I typical review flow would be done just like we're doing here, in GH, but you're right that this is raw markdown text, minus looks and feel. An alternative is reviewers could pull down your code from the pull request. 

But if this works for reviews, could you add a sentence, or a paragraph, before step 1., that explains it?

Reply:
done. thanks

Reply:
I don't personally see any benefit to us - anyone on our side who would be reviewing navigational changes already has a local environment setup. If big visual changes are made I usually include a screenshot.

I'm not 100% opposed to documenting this as option - but putting it at the same Heading level as the rest of this page gives the impression that it is how we expect users to make changes and get them reviewed. I'll defer to @dorbin on the final call, though.